ghcup-0.0.7.yaml

```diff
@@ -4407,6 +4406,87 @@ ghcupDownloads:
               dlHash: 79869dd72cb0a6f90f2e1e0e57af4ea6ee09b041550abc09252d1cf309b4f96e
+    3.10.2.0:
+      viPostInstall: "cabal run is currently partially broken on Windows, please see https://github.com/haskell/cabal/issues/9334"
+      viTags:
+        - Latest
+      viChangeLog: https://github.com/haskell/cabal/blob/master/release-notes/cabal-install-3.10.2.0.md
+      viArch:
+        A_64:
+          Linux_UnknownLinux:
+            unknown_versioning: &cabal-31020-64
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-alpine3_12.tar.xz
+              dlHash: b54e1cd235c47c62c03cdb9f6cf90e5fe8ae38c5e5befb9e21c8d1395f4bde05
+          Linux_Alpine:
+            unknown_versioning: *cabal-31020-64
+          Linux_CentOS:
+            unknown_versioning: &cabal-31020-64-centos7
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-centos7.tar.xz
+              dlHash: cfcdab399380dec7fedda55898bff975ac30b5d5d579433cbf8773b17c15f410
+          Linux_Debian:
+            ' ( == 9)': &cabal-31020-64-debian
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb9.tar.xz
+              dlHash: af5ce42114cf7720c37fee3238781df4c75bb74914c62e6a68833fb434e0cad7
+            ' ( == 10)':
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb10.tar.xz
+              dlHash: bdeb27c008b09c3b86f8a2768018d62a1aee02565304d123fda87ed432549418
+            ' ( >= 11)':
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-deb11.tar.xz
+              dlHash: 9ca5625c89e8fcada02edced5048c3a3db0254e2bef1eb792d549d633222b108
+            unknown_versioning: *cabal-31020-64-debian
+          Linux_Fedora:
+            '( >= 27 && < 33)': *cabal-31020-64-centos7
+            unknown_versioning:
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-fedora33.tar.xz
+              dlHash: 1e59dc1e1a1b33085a1789b8ddafb55026211454efe0b4e814c956ef86fe3ea5
+          Linux_Ubuntu:
+            '( < 19 )': &cabal-31020-64-ubuntu18
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-ubuntu18_04.tar.xz
+              dlHash: 7b0bb22c263ae0b43459de1995bf465560d412c12d47af011b2ac27b5d30c7aa
+            unknown_versioning: &cabal-31020-64-ubuntu20
+              dlUri: https://downloads.haskell.org/cabal/cabal-install-3.10.2.0/cabal-install-3.10.2.0-x86_64-linux-ubuntu20_04.tar.xz
+              dlHash: c2a8048caa3dbfe021d0212804f7f2faad4df1154f1ff52bd2f3c68c1d445fe1
+          Linux_Mint:
+            '< 20': *cabal-31020-64-ubuntu18
+            '>= 20': *cabal-31020-64-ubuntu20
```

Comment:
I realized we don't catch the case of "detected distro, but not it's version" as explained above. After we came to a conclusion, we should add the `unknown_version` case here.